roles/network_plugin/calico/defaults/main.yml

```diff
@@ -5,5 +5,6 @@ nat_outgoing: true
 # Use IP-over-IP encapsulation across hosts
 ipip: false
 
-# cloud_provider can only be set to 'gce' or 'aws'
-# cloud_provider:
+# Set to true if you want your calico cni binaries to overwrite the
+# ones from hyperkube while leaving other cni plugins intact.
+overwrite_hyperkube_cni: false
```

Comment:
Could we just overwrite the use_hyperkube_cni, which is by default is auto evaluated from the hcube version? Why to add a duplicating var? 👎